src/uikit.cs

```diff
@@ -7752,6 +8017,19 @@ interface UITargetedPreview : NSCopying {
 	[BaseType (typeof (NSObject))]
 	[DesignatedDefaultCtor]
 	partial interface UIImageAsset : NSSecureCoding {
+
+		[iOS (13,0), TV (13,0)]
+		[Export ("imageWithConfiguration:")]
+		UIImage FromConfiguration (UIImageConfiguration configuration);
```

Comment:
did we not chose `Create*` last time ?

Reply:
Unfortunately there are other `From` members and IIRC we agreed that create would be for new types to be consistent 

```
[Export ("imageWithTraitCollection:")]
UIImage FromTraitCollection (UITraitCollection traitCollection);
```